src/crypto/internal/boring/sig/sig_other.go

```diff
@@ -0,0 +1,17 @@
+// Package sig stubs internal/boringcrypto/sig
+package sig
+
+// BoringCrypto indicates that the BoringCrypto module is present.
+func BoringCrypto() {
+
```

Comment:
Should we panic here and in `FIPSOnly`?

Reply:
These functions aren't called; they're here so tooling can look for the body of the symbols and then figure out stuff about the binary: https://github.com/golang/go/blob/master/src/crypto/internal/boring/sig/sig_amd64.s#L7

Reply:
Yeah, this is what made me hesitate a bit with this patch. It breaks these signatures. But apparently it's only used for amd64 which makes me a bit less nervous about it (and amd64 isn't even the primary target of TinyGo).